components/list/list-item-mixin.js

```diff
@@ -483,15 +414,13 @@ export const ListItemMixin = superclass => class extends composeMixins(
 		}
 	}
 
-	disconnectedCallback() {
-		super.disconnectedCallback();
-		ro.unobserve(this);
-	}
+	/** TODO: remove this once usage of breakpoints on d2l-list-item has been moved to the list */
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
 
-	updated(changedProperties) {
-		super.updated(changedProperties);
-		if (changedProperties.has('breakpoints')) {
-			this.resizedCallback(this.offsetWidth);
+		if (changedProperties.has('breakpoints') && this.breakpoints) {
```

Comment:
This is the temporary code to make this change backwards compatible until the one usage of breakpoints is updated. This code only handles breakpoints being set initially (can move this to `updated` to make it a bit safer there - the usage does not seem to update it ever) and then once `breakpoints` is changed, it sets it on the parent list. This isn't the most performant, since for each list item it would be checking for the parent having `breakpoints`.